podcasts/End of Year/Stories/2025/PaidStoryWallView2025.swift

```diff
@@ -0,0 +1,104 @@
+import SwiftUI
+import AVKit
+import PocketCastsServer
+import PocketCastsUtils
+
+struct PaidStoryWallView2025: View {
+    let identifier = "plus_interstitial"
+
+    @StateObject private var model = PlusPricingInfoModel()
+
+    let subscriptionTier: SubscriptionTier
+
+    private let foregroundColor = Color.black
+
+    private let backgroundColor = Color(hex: "#96BCD1")
+
+    private let videoAspectRatio = CGFloat(1.37)
+
+    init(subscriptionTier: SubscriptionTier) {
+        self.subscriptionTier = subscriptionTier
+    }
+
+    var body: some View {
+        GeometryReader { geometry in
+            VStack(spacing: 0) {
+                Spacer()
+                    .frame(width: geometry.size.width)
+                    .background() {
+                        GeometryReader { geometry in
+                            HStack {
+                                Spacer()
+                                CustomVideoPlayerView(urlString: "playback_2025_plus")
+                                    .frame(width: ((geometry.size.height - 48.0) / videoAspectRatio).rounded(),
+                                           height: (geometry.size.height - 48.0))
+                                    .allowsHitTesting(false)
+                                Spacer()
+                            }
+                        }
+                    }
+                    .padding(.top, UIScreen.isSmallScreen ? 80 : 110)
+                    .allowsHitTesting(false)
+                StoryHeader2025(title: L10n.playback2025PlusUpsellTitle, description: L10n.playback2025PlusUpsellDescription, subscriptionTier: .plus, topPadding: 0)
+                Button(L10n.playback2025PlusUpsellButtonTitle) {
+                    guard let storiesViewController = SceneHelper.rootViewController() else {
+                        return
+                    }
+
+                    NavigationManager.sharedManager.showUpsellView(from: storiesViewController, source: .endOfYear, flow: SyncManager.isUserLoggedIn() ? .endOfYearUpsell : .endOfYear)
+                }
+                .buttonStyle(BasicButtonStyle(textColor: .black, backgroundColor: Color.clear, borderColor: .black))
+                .padding(.horizontal, 24)
+                .padding(.vertical, 6)
```

Comment:
The padding between text and button is bigger than(24). Considering the footer has 4 on each part in a vertical padding, we should add 20 here
```suggestion
                .padding(.top, 20)
                .padding(.bottom, 6)
```

Reply:
Fixed here: https://github.com/Automattic/pocket-casts-ios/pull/3669/commits/18e09f88bf4a5279dec644d9f2f558adafe55be6